.travis.yml

```diff
@@ -22,12 +22,13 @@ before_install:
   - if [[ $(npm -v | cut -d '.' -f 1) -lt 3 ]]; then npm i -g npm@^3; fi
 
 install:
-  - npm install --no-optional
+  - travis_retry npm install --no-optional
 
-script: npm run-script test-all:cover
+script:
+  - travis_retry npm run-script test-all:cover
```

Comment:
This one is failing consistently with ELIFECYCLE on our 0.12 Travis boxes. I'm willing to take this tradeoff right now. You can still monitor the primary test run log for failures if looking for the faster feedback. I'll plan to revert this on January 1.